scanners/http_observatory_scanner.py

```diff
@@ -13,12 +13,12 @@ def __init__(self, poll_interval=1):
     def scan(self, host):
         # Initiate the scan
         results = {}
-        analyze_url = self.api_url + '/analyze?host=' + host.targetname
+        analyze_url = self.api_url + '/analyze?host=' + host
```

Comment:
Seeing this concating, I wonder if we have validation on  self.api_url to ensure it doesn't contain a trailing slash

Reply:
Not in the code, as it is read from an env variable, but sure this can easily be implemented in the code.

Reply:
Added